app/controllers/api/v2/settings_controller.rb

```diff
@@ -22,7 +22,6 @@ class SettingsController < V2::BaseController
 
       api :GET, "/settings/", N_("List all settings")
       param_group :search_and_pagination, ::Api::V2::BaseController
-      add_scoped_search_description_for(Setting)
```

Comment:
Please, don't remove it completely, it's for documenting what can be used for searching via API/hammer.